etc/vagrant/etc-bash.bashrc

```diff
@@ -0,0 +1,68 @@
+# System-wide .bashrc file for interactive bash(1) shells.
```

Comment:
actually I don't fully understand the reason to store `.bashrc` files here at all. What's the reason, that we cannot use standard `.bashrc` provided by Vagrant box?